rktl_sim/src/simulator/car.py

```diff
@@ -69,31 +84,36 @@ def step(self, dt):
                 else:
                     self._psi -= self._STEERING_RATE * dt
 
-            # using bicycle model, extrapolate future state
+            # Extrapolate future state sing bicycle model.
             x_dot = self._v_rear * math.cos(theta + math.atan(math.tan(self._psi) / 2.0)) * \
-                math.sqrt(math.pow(math.tan(self._psi), 2.0) / 4.0 + 1.0)
+                    math.sqrt(math.pow(math.tan(self._psi), 2.0) / 4.0 + 1.0)
             y_dot = self._v_rear * math.sin(theta + math.atan(math.tan(self._psi) / 2.0)) * \
-                math.sqrt(math.pow(math.tan(self._psi), 2.0) / 4.0 + 1.0)
+                    math.sqrt(math.pow(math.tan(self._psi), 2.0) / 4.0 + 1.0)
             omega = self._v_rear * math.tan(self._psi) / self._LENGTH
         else:
             body_vel = self.cmd[0]
             if abs(body_vel) > self._MAX_SPEED:
                 body_vel = math.copysign(self._MAX_SPEED, body_vel)
 
-            body_curv = self.cmd[1]
-            if abs(body_curv) > self._MAX_CURVATURE:
-                body_curv = math.copysign(self._MAX_CURVATURE, body_curv) 
-            
+            body_curve = self.cmd[1]
```

Comment:
This is a misleading variable name. Previous variable name stands for curvature, indicating the inverse of a turning radius.

Reply:
I renamed it to body_curvature. Does this work, or should I keep it at body_curv?

Reply:
Either of those options should be fine

Reply:
W